__tests__/__unit__/uss/ZoweUSSNode.unit.test.ts

```diff
@@ -0,0 +1,617 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import * as vscode from "vscode";
+import * as zowe from "@zowe/cli";
+import * as imperative from "@zowe/imperative";
+import { Profiles, ValidProfileEnum } from "../../../src/Profiles";
+import { ZoweUSSNode } from "../../../src/uss/ZoweUSSNode";
+import { generateISession, generateISessionWithoutCredentials, generateIProfile,
+    generateFileResponse, generateTreeView, generateInstanceOfProfile } from "../../../__mocks__/generators/shared";
+import { generateUSSTree } from "../../../__mocks__/generators/uss";
+import * as fs from "fs";
+import * as path from "path";
+import { FAVORITE_CONTEXT, DS_BINARY_FILE_CONTEXT, DS_TEXT_FILE_CONTEXT, FAV_SUFFIX, USS_SESSION_CONTEXT,
```

Comment:
import * as globals? There's no sonar cloud anymore 😃 I can just imagine this import getting bigger as time goes by.

Reply:
Resolvedd